modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -46,7 +47,31 @@ const inlineAdditionalBehaviours = (editor: Editor, isStickyToolbar: boolean, is
           fadeOutClass: 'tox-dialog-dock-fadeout',
           transitionClass: 'tox-dialog-dock-transition'
         },
-        modes: [ 'top' ]
+        modes: [ 'top' ],
+        lazyViewport: (comp) => {
+          // If we don't have a special scrolling environment, then just use the default
+          // viewport of (window)
+          // TODO: Should we consult ui_of_tomorrow here also?
```

Comment:
- [ ] only execute when `ui_of_tomorrow: true`

Reply:
Actually, make a `ScrollingContext.detectWhenUiOfTomorrow` which takes `editor` as well and checks the option.